include/lsst/meas/extensions/shapeHSM/HsmMomentsControl.h

```diff
@@ -201,6 +202,15 @@ class HsmSourceMomentsAlgorithm : public HsmMomentsAlgorithm {
 };
 
 
+class HsmSourceMomentsRoundAlgorithm : public HsmSourceMomentsAlgorithm {
+public:
+    /// @brief Initialize with standard field names and customized documentation.
+    HsmSourceMomentsRoundAlgorithm(Control const & ctrl, std::string const & name, afw::table::Schema & schema,
+                                   char const* doc = "Source adaptive moments algorithm from HSM, with circular weight function") :
```

Comment:
As above, "Source adaptive moments via the HSM shape algorithm..." or something like that.